azure-pipelines.yml

```diff
@@ -17,7 +17,7 @@ pr:
 variables:
   # Cannot use key:value syntax in root defined variables
   - name: _TeamName
-    value: DotNetCore
+    value: dotnet-core-acquisition
```

Comment:
Question: is dotnet-core-acquisition the proper name for the templating repo?

Reply:
I used the same values as runtime team as requested in the issue #2324. Or did I misunderstand?

This is what runtime uses: https://github.com/dotnet/runtime/blob/49118f3385f12f87ec916ec3398a429fa3d11c69/eng/pipelines/runtime-official.yml#L27

Reply:
Looks like you're clear on the issue, I defer to you.